AENS.md

```diff
@@ -238,6 +272,98 @@ Note that `expire` is not an explicit message that is part
 of the protocol.
 
 
+### Protocol fees and protection times
```

Comment:
The fee in the table is missing units, as we discussed with @sennui it's 10^14 e.g. 3e14 - 5702887e14 (~570AE)